back-end/Models/comments/comment.js

```diff
@@ -0,0 +1,51 @@
+const connection = require("../../db");
+
+//Add new comment
+const addComment = (req, res) => {
+  const comment = req.body;
+  comment.user_id = null;
+  comment.post_id = null;
+  const query = `INSERT INTO comment (comment, thumbs_up) VALUES (?, ?)`;
+  const data = [comment.comment, comment.thumbs_up];
+  connection.query(query, data, (err, results) => {
+    if (err) throw err;
+    res.json(results);
+  });
+};
+
+//Update comment
+const updateCommentById = (req, res) => {
+  const token_user_id = res.token.user_id;

```

Comment:
do you mean `req.token` ?
Also, you could just name it so it uses camelcase instead and do that for the rest of the variables